indra/llappearance/llavatarappearance.cpp

```diff
@@ -799,7 +799,6 @@ void LLAvatarAppearance::buildCharacter()
     bool status = loadAvatar();
     stop_glerror();
 
-//  gPrintMessagesThisFrame = true;
```

Comment:
This gPringMessagesThisFrame global variable is legacy debug instrumentation that is not being used anymore.  I'm purging it to help keep the code more readable.  Fellow developers: please purge your personal debugging frameworks unless they are broadly useful and and come with some comments or other documentation.